traefik/sample-apps.go

```diff
@@ -38,3 +42,45 @@ func (mesh *Mesh) applyManifest(contents []byte, isDel bool, namespace string) e
 
 	return nil
 }
+
+// sidecarInjection enables/disables sidecar injection on a namespace
+func (mesh *Mesh) sidecarInjection(namespace string, del bool) error {
```

Comment:
Traefik mesh doesn't have side car proxies instead relies on configuring CoreDNS to offer custom routing. Please remove this function.

Reply:
Done